Rover/Rover.cpp

```diff
@@ -94,6 +94,9 @@ const AP_Scheduler::Task Rover::scheduler_tasks[] = {
     SCHED_TASK_CLASS(AP_ServoRelayEvents, &rover.ServoRelayEvents, update_events,  50,  200,  66),
 #if GRIPPER_ENABLED == ENABLED
     SCHED_TASK_CLASS(AP_Gripper,          &rover.g2.gripper,       update,         10,   75,  69),
+#if PRECISION_LANDING == ENABLED
+    SCHED_TASK(update_precland,      400,     50,  70),
```

Comment:
formatting

Reply:
That's how it is done at other places in the same file I think. Sorry, I couldn't get it. Can you please mention how should it be done?